master/Dockerfile.ubuntu

```diff
@@ -10,19 +12,29 @@ MAINTAINER  Buildbot maintainers
 # that everything is rebuilt
 ENV         security_updates_as_of 2016-07-30
 
+# This will make apt-get install without question
+ENV         DEBIAN_FRONTEND noninteractive
```

Comment:
I believe this is against Docker best practices, see https://github.com/docker/docker/issues/4032 and https://github.com/docker/docker/pull/4035